website/docs/guides/provider_reference.html.markdown

```diff
@@ -136,6 +136,15 @@ by precedence.
     the path of your service account key file in the
     `GOOGLE_APPLICATION_CREDENTIALS` environment variable, or configure
     authentication through one of the following;
+  
+     ```hcl
+     provider "google" {
+         alias       = "other"
```

Comment:
I don't think we want to add docs with an alias unless there's a particular reason to do so